docs/content/_docs/query_language.html

```diff
@@ -284,12 +242,31 @@ <h4>Operators</h4>
     <td><pre><code class="language-hql">&lt;a&gt; not in [&lt;b&gt;, ..]</code></pre></td>
     <td><pre><code class="language-json">["not", ["or", ["=", &lt;a&gt;, &lt;b&gt;], ..]]</code></pre></td>
   </tr>
+  
+  <tr>
+    <th scope="row">Tag True</th>
+    <td><pre><code class="language-hql">&lt;a&gt; true [&lt;b&gt;, ..]</code></pre></td>
+    <td><pre><code class="language-json">["true", &lt;a&gt;, &lt;b&gt;], ..]]</code></pre></td>
+  </tr>
 
+  <tr>
+    <th scope="row">Tag False</th>
+    <td><pre><code class="language-hql">&lt;a&gt; false [&lt;b&gt;, ..]</code></pre></td>
+    <td><pre><code class="language-json">["false", &lt;a&gt;, &lt;b&gt;], ..]]</code></pre></td>
+  </tr>
+  
   <tr>
     <th scope="row">RegEx On Tag</th>
     <td><pre><code class="language-hql">&lt;a&gt; ~ &lt;b&gt;</code></pre></td>
     <td><pre><code class="language-json">["~", &lt;a&gt;, &lt;b&gt;]</code></pre></td>
   </tr>
+  
+    <tr>
+    <th scope="row">Raw</th>
+    <td><pre><code class="language-hql">&lt;a&gt; q &lt;b&gt;</code></pre></td>
+    <td><pre><code class="language-json">["q", &lt;a&gt;, &lt;b&gt;]</code></pre></td>
```

Comment:
For `q`, isn't it a complex query and not a tag-operator? The examples here use `a` and `b` as tag placeholders, but I don't think that's what a query with `q` would look like.